packages/hint-valid-doctype/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "outDir": "dist"
```

Comment:
add the `strict: true`

Reply:
See https://github.com/webhintio/hint/issues/1400